plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -82,7 +83,7 @@ query_version = 2
 # include_query = []
 
 ## A list of queries to explicitly ignore.
-exclude_query = [ 'Schedulers' , 'SqlRequests']
+exclude_query = [ 'Schedulers' , 'SqlRequests','QueryStats']
```

Comment:
The issue with adding this to the exclude_query array is that people upgrading probably won't get the change, and so it'll start running by default for them. This is a problem in general with the exclude_query list being updated.

Reply:
If you upgrade the config file isn't overwritten so if you didn't have it excluded you will get it AFAIK

Reply:
yeah, I'm not too happy about upgrading and getting new queries suddenly. Can we exclude it by default and make you add it to the include_query list instead?

Reply:
not sure I get it, are you proposing to populate by both sections _include_queries_ and _exclude_queries_ of in the default configuration file?
I don't see any cons in that

Reply:
Well, I'm not sure I understand completely, but it seems like we get two different behaviors for new users to telegraf and upgrading users. New users get the query excluded and upgrading users get the query added. Almost seems like the opposite of what I would expect.

Reply:
I am not sure we should be adding everything "new" under include list, that also means every new person that adopts won't get it and have to explicitly include it correct? Certain things like Query level details given more expensive can be excluded by default or relegated to include list only but there has to be a decision on those types of queries. I like a good default queries that are lighter wait, and then include list for the rest if needed which are deemed heavier

Reply:
we are of the same mind there. Do you want to make any changes or are we good to merge?

Reply:
ok, now I get it.
Well the query itself can be somehow expensive, like the "sql_requests" one, that's why it has been excluded by default, if you want it, you will have to be explicit about it.
I don't see any solution to this kind of problem though (getting or not new queries on new setups or updates), I hope people read the release notes before upgrading.

Reply:
@ssoroka  the problem in V2 collector ( what we are trying to address with #7934  ) is t hat all queries are run by default.
@Trovalo  if ok we can close this what I propose is the following
a. We do not add this query at all for V2 (given it is heavy duty and is by default going to be run)
b. We do not add this for database_type by default given we don't want this to be run for default
c. A user actually uses include_list if wanted for this. I think Include_list as it stands today has a few issues ( I ran into when using that need to debug).

@Trovalo  are you ok either revisiting this after #7934  is merged or doing as part of it as a specific commit?

Reply:
I'm not aware of any issue with _include_query_, to me we can postpone this after #7934. (to keep things easier)
Solution "c" seems the best one to me, the point is how can we achieve that with given the 2 existing parameters, include and exclude.

For now, let's postpone it, we will see later what and how to do it.